spec/android_localize_helper_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper.rb'
+
+describe Fastlane::Helper::Android::LocalizeHelper do
+  describe 'get_library_version_from_gradle_config' do
+    it 'returns nil when gradle file is not present' do
+      allow(File).to receive(:exists?).and_return(false)
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key')).to be_nil
+    end
+
+    it 'returns nil when the key is not present' do
+      allow(File).to receive(:exists?).and_return(true)
+      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new('my-test-key = \'my_test_value\''))
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key2')).to be_nil
+    end
+
+    it 'returns the key content when the key is present' do
+      allow(File).to receive(:exists?).and_return(true)
+      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new('my-test-key = \'my_test_value\''))
```

Comment:
I think it should be worth testing with more dummy content than just a one-liner fake gradle file. Probably instead mock with content like this:

```
test-key-1 = 'foo'
oh-my-test-key = 'bad'
my-test-key   =  'good'
```

Also, what about when `my-test-key` uses double-quotes instead of simple-quotes? I'm guessing that so far our various `build.gradle` files in all our app repos might be using `fooVersion = 'x.y.z'` syntax… but I can easily imagine someone using double-quotes one day (especially when replacing the hash with a version or vice-versa and/if their IDE does some auto-lint/formatting and changes those)

Reply:
Good point! Let's update the regex to handle double quotes :-)